roles/openshift_openstack/tasks/check-prerequisites.yml

```diff
@@ -105,3 +105,19 @@
   - "{{ openshift_openstack_node_flavor }}"
   - "{{ openshift_openstack_lb_flavor }}"
   - "{{ openshift_openstack_etcd_flavor }}"
+
+- block:
```

Comment:
I think this is not a great way to do this.  Why not just use the os_volume module, users can specify volume name and volume id if they choose, that module will idempotently create the volume if necessary.